src/main/java/com/github/oehme/sobula/License.java

```diff
@@ -1,27 +1,67 @@
 package com.github.oehme.sobula;
 
+import java.util.regex.Pattern;
+
 public enum License {
 	EPL_V1_0(
 		"EPL-1.0", 
 		"Eclipse Public License, Version 1.0", 
-		"http://opensource.org/licenses/EPL-1.0",
-		"Eclipse Public License - v 1.0"
+		"https://www.eclipse.org/legal/epl-v10.html",
+		"Eclipse Public License\\s+-\\s+v 1.0"
 	), 
 	MIT(
 		"MIT", 
 		"The MIT License", 
-		"http://opensource.org/licenses/MIT"
+		"http://opensource.org/licenses/MIT",
+		Pattern.quote(
+			"Permission is hereby granted, free of charge, to any person obtaining a copy\n" +
```

Comment:
As I figured out, the MIT license is not a real license like the EPL or the GPL-like licenses.
It is more a license template. So, you will not find any specific hint in such a particular real license that it was copied from the MIT license template. The only way to recognize it is to check for the whole text (in that sense I should really copy in the while license template text in ...).